### DISCUSSION TITLE
Option to keep original From display name.

### DISCUSSION THREAD

**Discussion by yo1dog** [anonaddy/anonaddy#470] [Feature Requests]
Would love to be able to disable the appending of the sender's email to the From mailbox display name and have the display name passed through unmodified.

Currently, the From mailbox display name is modified to include the email of the original sender, and there is no way to change this behavior (that I know of):
https://github.com/anonaddy/anonaddy/blob/4f47bc4f4cf15e20555e99290e09cc85e82691ba/app/Mail/ForwardEmail.php#L157

Example:
```
From                           : "Fu Bar 'fubar123 at gmail.com'" <alias@customdomain.com>
X-AnonAddy-Original-From-Header: "Fu Bar" <fubar123@gmail.com>
```

In my opinion, this makes it too noisy and difficult to visually scan my inbox:

- **PayPal 'noreply at paypal.com'** - You have a new request for...
- **LocalBank 'customer-service at email.localbank.com'** - Your deposit has been...
- **Magazine Monthly 'nobody at magazine-mailinglist.emailsender.com'** - The newest issue of...

vs the original:

- **PayPal** - You have a new request for...
- **LocalBank** - Your deposit has been...
- **Magazine Monthly** - The newest issue of...

**Comment by willbrowningme** [comment]
Thanks, I will be adding this as a configurable option shortly

**Reply by yo1dog** [reply]
Sweet thanks. Let me know if I can help at all.

**Reply by yo1dog** [reply]
Any update on this? Thanks!

**Reply by willbrowningme** [reply]
Will be live by the end of this month.
